### ISSUE TITLE
Assume role credentials are not masked by Azure DevOps when printed

### ISSUE THREAD

**Issue by wm999** [aws/aws-toolkit-azure-devops#399]
**Describe the bug**

Hello,

Not sure if this an intended behaviour or a peculiarity of Azure Dev Ops or something I am doing wrong. Essentially AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY are masked if echoed - when the service connection has NO "role to assume". If you set the "role to assume" in the service connection, they are not masked if you echo. Noting these are the temporary credentials at this point.

**To reproduce**

Scenario 1 - Service Connection defined with credentials defined with NO assume role.

Make sure role to assume in the service connection is EMPTY i.e not set.

Run below (adjusting for your credential definition) and masking occurs i.e echo just shows ***

Sorry not sure how to fix the format below
```
# Starter pipeline

# Start with a minimal pipeline that you can customize to build and deploy your code.
# Add steps that build, run tests, deploy, and more:
# https://aka.ms/yaml

trigger:
- main

pool:
  vmImage: ubuntu-latest


steps:

- task: AmazonWebServices.aws-vsts-tools.AWSShellScript.AWSShellScript@1
  displayName: 'test1'
  inputs:
    awsCredentials: 'aws-sandpit'
    regionName: 'ap-southeast-2'
    scriptType: inline
    inlineScript: |
     echo "$AWS_ACCESS_KEY_ID"
     echo "$AWS_SECRET_ACCESS_KEY"
     aws s3 ls
```
Scenario 2 - Same service connection but role to assume added to the service connection definition and same pipeline

In service connection add a valid assume role ARN.
Rerun the job using the same pipeline
Values are not masked and are actual values which appear to be the temporary credentials.

**Expected behavior**

May not be bug, but assumed masking would still occur. I guess Azure DevOps is looking for the value and not the variable.

**Screenshots**

**Your Environment**

-   Azure Devops cloud
-   AWS Toolkit for Azure DevOps version: ? latest

**Comment by hunterwerlla** [comment]
Since assume role gets temporary credentials through `sts assume role` access_key and secret_key will be different from the saved ones. Azure DevOps makes an effort to not print secrets, but cannot help in cases like this.

**Comment by hunterwerlla** [comment]
This was released as part of today's 1.10 release
